binary-prediction/app/src/styles.css

```
@@ -0,0 +1,619 @@
+  --tick-red: #ff6b6b;
+  --oracle-cyan: #68d8ff;
+  --expiry-amber: #f4c55f;
+  --shadow: 0 24px 80px rgba(0, 0, 0, 0.35);
+  font-family:
+    Inter,
+    ui-sans-serif,
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)